packages/core/src/App/Components/Elements/VerticalTabs/vertical-tab-content-container.jsx

```diff
@@ -51,7 +51,6 @@ class VerticalTabContentContainer extends React.PureComponent {
                 <div className={classNames('vertical-tab__content-container', tab_container_classname)}>
                     { is_routed ?
                         <Switch>
-                            <Redirect exact from={routes.reports} to={routes.positions} />
```

Comment:
isn't this setting the default routes for reports?, is it okay to be removed?

Reply:
it was moved to routes-with-subroutes so we don't hardcode anymore so that it works for others like cashier as well.